docs/docs-content/clusters/pcg/manage-pcg/configure-proxy.md

```diff
@@ -0,0 +1,208 @@
+  steps. Below is a list of helpful links to set up proxy configurations for some common Kubernetes platforms:
+
+  - [Amazon EKS](https://repost.aws/knowledge-center/eks-http-proxy-containerd-automation)
+  - [Azure AKS](https://learn.microsoft.com/en-us/azure/aks/http-proxy)
+  - [vSphere](https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-vcenter-configuration/GUID-7576FBBE-8B67-4D3F-B859-C9688E9442D7.html)
+
+    :::warning
+
+    This feature has only been tested on AWS, Azure, and VMware vSphere. If you are using a different cloud provider,
+    contact our support team for additional guidance.
+
+    :::
+
+- Download the Reach Helm Chart provided by our support team. The Reach Helm Chart is used to deploy the Reach service
+  into the Kubernetes cluster where the PCG is deployed. The Reach service is used to manage proxy configurations for
+  the PCG. Contact our support team to obtain the Reach Helm Chart.
+
+- `cert-manager` is not already deployed in the Kubernetes cluster where the PCG is deployed. If `cert-manager` is
+  already deployed, make sure you edit the deployment to use the
+  `- --feature-gates=AdditionalCertificateOutputFormats=true` flag. Otherwise, the Reach service cannot manage the proxy
+  configurations.
+
+## Enable Proxy
+
+1. Open a terminal session and navigate to the folder where you downloaded the Reach Helm Chart zip file.
+
+2. Unzip the downloaded artifact you received from the support team.
+
+   ```shell
+   unzip release-*.zip -d palette
+   ```
+
+3. Navigate to the release folder inside the unzipped folder.
+
+   ```shell
+   cd palette/charts/release-*/
+   ```
+
+4. Deploy `cert-manager` to the Kubernetes cluster where the PCG is deployed. Use the following command to deploy the
+   `cert-manager` Helm Chart.
+
+   ```shell
+   helm upgrade --values extras/cert-manager/values.yaml \
+   cert-manager extras/cert-manager/cert-manager-*.tgz --install
+   ```
+
+5. Next, navigate to the reach-system folder and extract the Reach Helm Chart.
+
+   ```shell
+   cd extras/reach-system/ && tar -xvzf reach-system-*.tgz
+   ```
+
+6. Use a text editor and open the **values.yaml** file that is inside the **reach-system** directory. Fill out the
+   following YAML fields with the proxy configuration details:
+
+   - `reachSystem.enabled`: Set this field to `true` to enable the Reach service.
+   - `reachSystem.proxySettings.http_proxy`: The HTTP proxy URL, including the port number.
+   - `reachSystem.proxySettings.https_proxy`: The HTTPS proxy URL, including the port number.
+   - `reachSystem.proxySettings.no_proxy`: A comma-separated list of URLs that should bypass the proxy.
+   - `reachSystem.proxySettings.ca_crt_path`: The path to the CA certificate file used to authenticate the proxy server.
+     Make sure the CA is in PEM format. If you do not have a CA certificate, leave this field empty.
```

Comment:
I think this should be a table, to stay consistent with the other instances where we instruct users to fill out YAMLs 🤔